prov/efa/Makefile.include

```diff
@@ -82,6 +82,7 @@ _efa_headers = \
 
 if ENABLE_EFA_UNIT_TEST
 check_PROGRAMS += prov/efa/test/efa_unit_test
```

Comment:
Do we still need this the `efa_unit_test` in `check_PROGRAMS`?

Reply:
done